tests/integration/test_containerd_integration.py

```diff
@@ -13,6 +18,11 @@
 log = logging.getLogger(__name__)
 
 
+def K_cmd(cmd: str) -> str:
+    """Return a kubectl command with the kubeconfig path."""
+    return f"kubectl --kubeconfig /root/.kube/config {cmd}"
```

Comment:
Just a suggestion, feel free to ignore it:
```suggestion
def format_kubectl_cmd(cmd: str) -> str:
    """Return a kubectl command with the kubeconfig path."""
    return f"kubectl --kubeconfig /root/.kube/config {shlex.split(cmd)}"
```